sdk/src/client/api/block_builder/pow.rs

```diff
@@ -8,11 +8,11 @@ use crate::pow::miner::{Miner, MinerBuilder, MinerCancel};
 #[cfg(target_family = "wasm")]
 use crate::pow::wasm_miner::{SingleThreadedMiner, SingleThreadedMinerBuilder};
 use crate::{
-    client::{ClientInner, Error, Result},
+    client::{Client, Error, Result},
     types::block::{parent::Parents, payload::Payload, Block, BlockBuilder, Error as BlockError},
 };
 
-impl ClientInner {
+impl Client {
```

Comment:
Just making sure, this is not breaking in any way since Client derefs to ClientInner, right?

Reply:
I will think about alternative impls though. I'm not 100% with this one.